### PR TITLE
Overhaul Teams reference overview to include all SDKs/libraries

### PR DESCRIPTION
**Pull request by mobajemu-msft**

### CODE REVIEW

**Comment by mobajemu-msft** [comment]
@erikadoyle @JoshuaPartlow

File: nodejs/overview/msteams-client.md [hunk 1]
```diff
@@ -1,61 +1,111 @@
 ---
-title: Microsoft Teams JavaScript client library
+title: Microsoft Teams Client API reference
 description: Reference documentation for the latest Microsoft Teams JavaScript client library
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:3]: Let's generalize this title to include all the libraries in the TOC. How about:

title: Microsoft Teams SDK reference
description: Reference documentation for Microsoft Teams SDK

(Using "SDK" in the holistic sense to include all the API libraries listed in the doc)

File: nodejs/overview/msteams-client.md [hunk 2]
```diff
@@ -1,61 +1,111 @@
...
-keywords: msteams teams client sdk javascript library reference latest
+ms.date: 01/03/2023
+ms.topic: reference
+keywords: msteams live-share teamsfx teams client sdk javascript library reference latest
 ---
-# Microsoft Teams JavaScript client library
+# Microsoft Teams Client API reference
 
-Starting with *@microsoft/teams-js* version `2.0.0`, the Microsoft Teams JavaScript client library has been refactored to enable [Teams apps to run in Outlook and Office](/microsoftteams/platform/m365-apps/overview), in addition to Microsoft Teams. From a functional perspective, the TeamsJS SDK is a superset of TeamsJS 1.x versions: it supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Office.
+This page contains general information about the client library to support you in the development of using The Microsoft Teams Client SDK.
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:10]: Also generalize the description here (Teams Client SDK implies only TeamsJS, as that's what it used to be called)

File: nodejs/overview/msteams-client.md [hunk 3]
```diff
@@ -1,61 +1,111 @@
...
 
-> [!TIP]
-> Refer to the [Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library) overview for guidance on building both Teams-only and Microsoft 365-enabled apps and for current SDK capability support across Teams, Outlook, and Office.
+| Library | Description | APIs |
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:12]: The "APIs" column label feels a little misleading, like it links to the actual code rather than reference documentation.

File: nodejs/overview/msteams-client.md [hunk 4]
```diff
@@ -1,61 +1,111 @@
...
+| **TeamsJS** | Teams Java script client library enables developers to create app content hosted in [IFrame](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) across the Microsoft 365 ecosystem | [TeamsJS ](../docs-ref-autogen/%40microsoft/teams-js/index.yml) |
+| **TeamsFx** | TeamsFx helps to reduce tasks by using Microsoft Teams single sign-on (SSO) and accessing cloud resources down to single line statements with little configuration | [Teamsfx](../docs-ref-autogen/%40microsoft/teamsfx/index.yml) |
 
-## Finding the library
+## Microsoft Teams Live Share
 
-The TeamsJS client is distributed as an npm package. The latest version can be found here:
+The [Live share SDK]( https://github.com/microsoft/live-share-sdk) is enabled to allow for collaborative moments across Microsoft Teams through `sidePanel` and `meetingStage`, media synchronization for HTML `<video>` and `<audio>` elements, and annotation tools for whiteboard and PowerPoint presentations.
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:20]: It would be great to explain here the relationship between the three LiveShare libraries here--which one(s) you would use for different scenarios. Also is LiveShare a superset of LiveShare Canvas + LiveShare Media?

**Review by erikadoyle — COMMENTED** [review]
Great start! It would also be useful to provide a kind of "next steps" link (for example, to a get started tutorial or the [overview](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-library) in the case of TeamsJS) for each of these sections.

Also don't forget to fix the validation warnings :)

File: nodejs/overview/msteams-client.md (hunk 3 above)
Comment (mobajemu-msft) [reply nodejs/overview/msteams-client.md:12]: would 'Code' be a better title?

File: nodejs/overview/msteams-client.md (hunk 2 above)
Comment (JoshuaPartlow) [reply nodejs/overview/msteams-client.md:10]: I feel like at some point I heard guidance to say article vs page. So maybe do something like "This article provides you with general information about the libraries that Microsoft offers to support you in the development of..."

File: nodejs/overview/msteams-client.md (hunk 3 above)
Comment (JoshuaPartlow) [reply nodejs/overview/msteams-client.md:12]: If it's linking to the reference docs, maybe just say "Docs" or "Reference"

Comment (JoshuaPartlow) [reply nodejs/overview/msteams-client.md:12]: Also, maybe just me, but I'd almost consider turning this into an intro paragraph instead. Set the stage for why these are grouped and just have the links be off the library name keyword. Can help with suggestion if you'd like to talk about how to do that. Then the following sections can be more details about the individual library.

File: nodejs/overview/msteams-client.md (hunk 4 above)
Comment (JoshuaPartlow) [reply nodejs/overview/msteams-client.md:20]: "...is enabled to allow for collaborative moments across Microsoft teams..." is a weird sentence to me. I can see a library enabling scenarios, but not the library itself being enabled if that makes sense. "...enables developers to create collaborative moments..." maybe?

File: nodejs/overview/msteams-client.md [hunk 5]
```diff
@@ -1,61 +1,111 @@
...
+## Microsoft Teams Live Share
 
-The TeamsJS client is distributed as an npm package. The latest version can be found here:
+The [Live share SDK]( https://github.com/microsoft/live-share-sdk) is enabled to allow for collaborative moments across Microsoft Teams through `sidePanel` and `meetingStage`, media synchronization for HTML `<video>` and `<audio>` elements, and annotation tools for whiteboard and PowerPoint presentations. 
 
-www.npmjs.com/package/@microsoft/teams-js
+### Finding the library
 
-## Installing the library
+The Live share SDK is a JavaScript package comprised of three sub-packages ([Live Share](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md), [Live Share media](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-media-capabilities.md), and [Live Share canvas](msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md)) which is published on npm and can be installed using npm or yarn.
```
Comment (JoshuaPartlow) [inline nodejs/overview/msteams-client.md:24]: I don't think "which is" works in the current sentence ordering. You could re-order if it's the main package you want to talk about or if it's the sub-packages then you'd use "are" instead.

The Live Share SDK, which is published on NPM, is a JavaScript package comprised of....
The Live Share SDK is a JavaScript package, comprised of three sub-packages (....), which is....

 Also should that "Live share" be "Live Share" ?

File: nodejs/overview/msteams-client.md [hunk 6]
```diff
@@ -1,61 +1,111 @@
...
 
-You can install the package using npm or yarn:
+#### Live share
 
-* `npm install --save @microsoft/teams-js`
-* `yarn add @microsoft/teams-js`
+To install the latest version of the Live share package to your application:
```
Comment (JoshuaPartlow) [inline nodejs/overview/msteams-client.md:28]: Maybe consider summarizing and then Tab the installs to make more condensed or group based on npm vs yarn? Depends a bit on if you'd install these standalone or always all together.

File: nodejs/overview/msteams-client.md [hunk 7]
```diff
@@ -1,61 +1,111 @@
...
+
+```bash
+yarn add @microsoft/live-share@next
+yarn add @microsoft/live-share-canvas@next
 ```
 
-You can also reference the entire library in html pages using a script tag.  There are three ways to do this:
+## Microsoft Teams JavaScript client library
+
+The Microsoft Teams JavaScript client library version `2.x.x` is enabled [run apps across Microsoft 365](/microsoftteams/platform/m365-apps/overview). The TeamsJS SDK is a superset of TeamsJS `1.x` versions: and supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Microsoft 365. Refer to the [Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library) overview for guidance on building Microsoft 365-enabled apps.
```
Comment (JoshuaPartlow) [inline nodejs/overview/msteams-client.md:81]: The "is enabled" text is still odd to me. I'd suggest rewording. This also just differentiates between v2 and v1, it doesn't really introduce the client library. Maybe a couple sentences before discussing the versions.

File: nodejs/overview/msteams-client.md [hunk 8]
```diff
@@ -1,61 +1,111 @@
...
+OR
 
-<!-- Microsoft Teams JavaScript API (via local) -->
-<script src="MicrosoftTeams.min.js"></script>
+To install using yarn
+
+```bash
+yarn add @microsoft/teams-js`
 ```
+## Microsoft Teams Fx library
```
Comment (JoshuaPartlow) [inline nodejs/overview/msteams-client.md:99]: You might consider having an overarching section about MS Teams that introduces both these libraries, then have subsections that discuss them (and their installation) individually.

File: nodejs/overview/msteams-client.md (hunk 3 above)
Comment (erikadoyle) [reply nodejs/overview/msteams-client.md:12]: For column label, how about "API reference"?

Also agree with Josh on the description--would be great to flesh it out into a full paragraph describing the different libraries that comprise Teams SDK.

**Comment by erikadoyle** [comment]
@surbhigupta12 could someone on your team help to review these revisions for Teams [SDK Reference](https://learn.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest) overview? (Goal is to introduce all the various Teams libraries/SDKs, rather than only TeamsJS). Thank you!

File: nodejs/overview/msteams-client.md [hunk 9]
```diff
@@ -1,61 +1,110 @@
...
+description: Reference documentation for Microsoft Teams SDK
+ms.date: 01/03/2023
+ms.topic: reference
+keywords: msteams live-share teamsfx teams client sdk javascript library reference latest
 ---
-# Microsoft Teams JavaScript client library
+# Microsoft Teams SDK reference
 
-Starting with *@microsoft/teams-js* version `2.0.0`, the Microsoft Teams JavaScript client library has been refactored to enable [Teams apps to run in Outlook and Office](/microsoftteams/platform/m365-apps/overview), in addition to Microsoft Teams. From a functional perspective, the TeamsJS SDK is a superset of TeamsJS 1.x versions: it supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Office.
+This article provides you with general information about the libraries that Microsoft offers to support you in the development of using The Microsoft Teams SDK.
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:10]: How about:

There are a number of JavaScript/TypeScript libraries available to support various aspects of Teams app development. This article provides an overview of each, along with basic installation instructions.

File: nodejs/overview/msteams-client.md [hunk 10]
```diff
@@ -1,61 +1,110 @@
...
 
-> [!TIP]
-> Refer to the [Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library) overview for guidance on building both Teams-only and Microsoft 365-enabled apps and for current SDK capability support across Teams, Outlook, and Office.
+| Library | Description | API reference |
+|-|-|-|
+| **Live Share** | Live Share transforms the Microsoft 365 platforms to be more a collaborative experience without writing any dedicated back-end code | [Live Share](../docs-ref-autogen/%40microsoft/live-share/index.yml)<br/><br/>[Live Share canvas](../docs-ref-autogen/%40microsoft/live-share-canvas/index.yml)<br/><br/>[Live Share media](../docs-ref-autogen/%40microsoft/live-share-media/index.yml) |
+| **TeamsJs** | Teams Java script client library enables developers to create app content hosted in [IFrame](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) across the Microsoft 365 ecosystem | [TeamsJS ](../docs-ref-autogen/%40microsoft/teams-js/index.yml) |
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:15]: Teams Java script client library -> TeamsJS client library

File: nodejs/overview/msteams-client.md [hunk 11]
```diff
@@ -1,61 +1,110 @@
...
+# [Yarn](#tab/yarn)
+
+To install using yarn
+```bash
+yarn add @microsoft/live-share@next
+```
+
+---
+
+#### Live Share media
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:45]: Add brief intro here. Consider using [this description from docs](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-live-share-media-capabilitie):

The Live Share SDK enables robust media synchronization for any HTML <video> and <audio> element with just a few lines of code. By synchronizing media at the player state and transport controls layer, you can individually attribute views and license, while providing the highest possible quality available through your app.

File: nodejs/overview/msteams-client.md [hunk 12]
```diff
@@ -1,61 +1,110 @@
...
 
-## Installing the library
+The Live Share SDK is a JavaScript package comprised of three sub-packages ([Live Share](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md), [Live Share media](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-media-capabilities.md), and [Live Share canvas](/msteams-docs/msteams-platform/apps-in-teams-meetings/teams-live-share-canvas.md)) which are published on npm and can be installed using npm or yarn.
 
-You can install the package using npm or yarn:
+To install the latest version of the Live Share packages to your application:
 
-* `npm install --save @microsoft/teams-js`
-* `yarn add @microsoft/teams-js`
+#### Live Share core
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:28]: Add brief description. For example: 

The Live Share SDK can be added to your meeting extension's `sidePanel` and `meetingStage` contexts with minimal effort to unlock new collaborative app scenarios throughout the lifecycle of a meeting. For more, see [Live Share core capabilities](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-live-share-capabilities)

File: nodejs/overview/msteams-client.md [hunk 13]
```diff
@@ -1,61 +1,110 @@
...
+yarn add @microsoft/live-share@next
+yarn add @microsoft/live-share-media@next
 ```
 
-You can also reference the entire library in html pages using a script tag.  There are three ways to do this:
+---
 
-> [!IMPORTANT]
-> Do not copy/paste these `<script src=...` URLs from this page; they refer to a specific version of the SDK. To get the `<script src=...></script>` markup for the latest version, always go to https://www.npmjs.com/package/@microsoft/teams-js.
+#### Live Share canvas
```
Comment (erikadoyle) [inline nodejs/overview/msteams-client.md:61]: Suggestion for description:

Using Live Share canvas, your app can bring the power of [PowerPoint Live](https://support.microsoft.com/en-us/office/present-from-powerpoint-live-in-microsoft-teams-28b20e74-7165-499c-9bd4-0ad975d448ad) inking tools with minimal effort. For more, see [Live Share canvas overview](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-live-share-canvas)

**Comment by erikadoyle** [comment]
Looking good @mobajemu-msft -- just a few last issues then I'm ready to sign off.
